tests/functions/test_move.py

```diff
@@ -184,6 +184,7 @@ def test_move_unique():
         df.move(source="a", axis=1)
 
 
+@pytest.mark.xfail(reason="allow MultiIndex move")
```

Comment:
@samukweku can you clarify the reason a bit more please?

The way I'm reading this reason is: "we expect the test to fail because allow MultiIndex move", and I'm not getting why "allow MultiIndex move" is the reason for allowing the test to fail. Is it that we expect the test to fail because MultiIndex moving is allowed?

Other than that, thank you for doing maintenance on the rest of the codebase here.

Reply:
my bad. `move` supports MultiIndex. I should give it a better description